bindata/assets/alerts/client-go-requests.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: client-go
+  namespace: openshift-kube-apiserver
+spec:
+  groups:
+    - name: control-plane-client-go
+      rules:
+        # latency by:
+        # - job (kubelet, apiserver. controller-manager, ...)
+        # - destination host: internal-lb, kubernetes.default service, direct connection ...
+        - record: cluster:rest_client_request_duration_seconds:internal_load_balancer:histogram_quantile
+          labels:
+            quantile: "0.99"
+          expr: |
+            histogram_quantile(0.99,
+              sum(rate(
+                label_replace(rest_client_request_duration_seconds_bucket{verb="GET",url=~"https?://api-int.*"},
+                  "host","$1","url","https?://([^/\\s]+).*")[5m:30s]
+                )
+              ) by (le,host,service,namespace,node)
+            )
```

Comment:
![image](https://user-images.githubusercontent.com/6450081/146750327-04afc8cb-32d3-4609-b776-c4ff5e4af42a.png)

Reply:
this is a great view of the internal load balancer , we can also differentiate by kubelet and node

Reply:
Yes, this recording rule is great to see the latency of the internal load balancer. The only thing I would change for telemetry purposes is: `by (le,host,service,namespace,node)`. As you mentioned it is great to differentiate kubelet and other services, but we can't afford this granularity in telemetry as each additional serie is expensive to store. My suggestion would be to keep this granularity for the dashboards and alerts, but reduce it to `by (le)` for telemetry so that we can reduce the number of metrics we push for the internal load balancer latency to 1.